features/openhab-addons-external/src/main/resources/conf/voicerss.cfg

```diff
@@ -1,5 +0,0 @@
-######################## VoiceRSS Text-to-Speech Engine ########################
-
-# API key you got when you subscribed to the Voice RSS service
-# Open your profile to get the value: http://www.voicerss.org/personel/
-#apiKey=1234567890
```

Comment:
why is this class removed?

Reply:
This is part of the 2.5.x addons, which is auto-ported to 3.0.